### DISCUSSION TITLE
is byte-buddy & agent FIPS compliant?

### DISCUSSION THREAD

**Discussion by tanwani-mohit** [raphw/byte-buddy#1606] [Q&A] [answered]
We are using byte-buddy in our project, and we have to make sure that all dependancies are FIPS complaint or not. Thanks in advance!

**Comment by raphw** [comment, marked as answer]
Byte Buddy does not implement encryption or store data, so it does not apply. This also means it's compliant.

**Reply by tanwani-mohit** [reply]
Thank you so much for answering :)
